src/mlpack/methods/reinforcement_learning/environment/acrobot.hpp

```diff
@@ -110,10 +118,9 @@ class Acrobot
    * @param maxVel2 The max angular velocity of link2.
    * @param dt The differential value.
    * @param doneReward The reward recieved by the agent on success.
-   * @param maxSteps The number of steps after which the episode
-   *    terminates. If the value is 0, there is no limit.
    */
-  Acrobot(const double gravity = 9.81,
+  Acrobot(const size_t maxSteps = 500,
```

Comment:
I think doing this will also break backwards compatitibility?

Reply:
I think that keeping a StepLimit in training config is of no use since its already present in the form of `maxSteps` in each env. And to make it more accessible, I've added it as a first parameter.

Reply:
Personally I like to make the RL more simple, even if that mean we will break compatitibility with older versions.